include/miopen/miopen.h

```diff
@@ -583,6 +583,50 @@ typedef enum
  */
 MIOPEN_EXPORT miopenStatus_t miopenCreateTensorDescriptor(miopenTensorDescriptor_t* tensorDesc);
 
+/*! @brief Set the quantization scale of Tensor Descriptor
+ *
+ * API for setting the quantization scale of tensor descriptor.
+ * @param tensorDesc  Pointer to a tensor descriptor type
+ * @param quantScales Quantization scales
+ * @param len         Length of quantization scales buffer
+ * @return            miopenStatus_t
+ */
+MIOPEN_EXPORT miopenStatus_t miopenSetQuantizationScales(miopenTensorDescriptor_t tensorDesc,
+                                                         double* quantScales,
+                                                         size_t len);
+
+/*! @brief Set the quantization bias of Tensor Descriptor
+ *
+ * API for setting the quantization bias of tensor descriptor.
+ * @param tensorDesc  Pointer to a tensor descriptor type
+ * @param quantBiases Quantization biases
+ * @param len         Length of quantization biases buffer
+ * @return            miopenStatus_t
+ */
+MIOPEN_EXPORT miopenStatus_t miopenSetQuantizationBiases(miopenTensorDescriptor_t tensorDesc,
+                                                         double* quantBiases,
```

Comment:
How does MIOpen want to kernel to compute with the precision level? I see double precision as input argument but realistically, is this really necessary?

I'm guessing this shall be fp32 (or needs to be converted to fp32), such that int32 convolution result can cast up also to fp32 for computation.